### PR TITLE
Fix/0318night

### PR DESCRIPTION
**Pull request by Zhenkun0**
finished the process of notification and chat. add the payment and show trainee of trainer

### CODE REVIEW

File: base/models.py [hunk 1]
```diff
@@ -39,7 +39,17 @@ class Trainer(models.Model):
         blank=True,
         related_name='trainer'
     )
-    image = models.TextField(null=True, blank=True)
```
Comment (bonn1112) [inline base/models.py:42]: You should not change the model directly. Please use the migration file.

File: base/serializers.py [hunk 2]
```diff
@@ -98,6 +98,24 @@ class Meta:
...
+        fields = '__all__'
+
+
+class NoteSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Note
+        fields = '__all__'
+
+
+class PaymentSerializer(serializers.ModelSerializer):
```
Comment (bonn1112) [inline base/serializers.py:113]: why are you creating Payment??
We should use Order table.

Comment (Zhenkun0) [reply base/serializers.py:113]: Oh, the frontend want us to write this part. to show three payment plan.

Comment (bonn1112) [reply base/serializers.py:113]: what about just adding fields to the order table ??

Comment (bonn1112) [reply base/serializers.py:113]: It is strange to have both an order table and a payment table.

File: base/views/users.py [hunk 3]
```diff
@@ -195,3 +205,96 @@ def updateTrainer(request, pk):
...
+
+
+@swagger_auto_schema(methods=['get'], responses={200: trainees_response})
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def getMyAcceptedTrainees(request):
+    trainer_id = request.user.trainer.pk
+    obj = Chat.objects.filter(isAccepted=True).filter(trainer=trainer_id).values('trainee')
+    list_trainees = list(set([ i['trainee'] for i in obj ]))
+    print(list_trainees)
```
Comment (bonn1112) [inline base/views/users.py:287]: plz delete debug

**Pull request merged**